.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
That's all you need now? Maybe better to use https://github.com/github/gitignore/blob/master/Node.gitignore anyway?